packages/core/variables/_index.scss

```diff
@@ -1,6 +1,7 @@
 @forward "breakpoints";
 @forward "colors" as color-* hide $varslingsfarger, varslingsfarge;
 @forward "shadow";
+@forward "new-shadow" as shadow-*;
```

Comment:
Kan vi kalle fila `_shadows` i stedet? Om vi en dag rydder vekk `_shadow` er det greit å ha et navn som gir mening på de skyggene vi ønsker å beholde, uten å måtte gjøre en breaking change for de som bruker riktige verdier (riktignok på feil måte om de importerer fila, men for dét :P).